e2e/pages/events/addHearingBookingDetailsEvent.page.js

```diff
@@ -0,0 +1,57 @@
+const I = actor();
+
+module.exports = {
+  fields: function (index) {
+    return {
+      hearingBooking: {
+        hearingType: {
+          caseManagement: `#hearingDetails_${index}_hearingType-CASE_MANAGEMENT`,
+        },
+        hearingVenue: `#hearingDetails_${index}_hearingVenue`,
+        date: {
+          day: `#hearingDetails_${index}_hearingDate-day`,
+          month: `#hearingDetails_${index}_hearingDate-month`,
+          year: `#hearingDetails_${index}_hearingDate-year`,
+        },
+        preHearingAttendance: `#hearingDetails_${index}_preHearingAttendance`,
+        hearingTime: `#hearingDetails_${index}_hearingTime`,
+        hearingNeedsBooked: {
+          interpreter: `#hearingDetails_${index}_hearingNeededDetails-INTERPRETER`,
+          welsh: `#hearingDetails_${index}_hearingNeededDetails-SPOKEN_OR_WRITTEN_WELSH`,
+          somethingElse: `#hearingDetails_${index}_hearingNeededDetails-SOMETHING_ELSE`,
+        },
+        giveDetails: `#hearingDetails_${index}_hearingNeededGiveDetails`,
+        judgeTitle: `#hearingDetails_${index}_judgeTitle`,
+        judgeFullName: `#hearingDetails_${index}_judgeFullName`,
+      },
+    };
+  },
+
+  addHearingButton: '//*[@id="hearingDetails"]/div/button[1]',
```

Comment:
this looks weird

Reply:
its the locator of the element, done the same as the other buttons